.github/workflows/AssignCommitter.yml

```
@@ -0,0 +1,47 @@
+name: Assign Committer
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  assign-reviewer:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.draft != true }}
```

We don't want this action to run in people's forks of Julia:

```suggestion
    if: ${{ github.repository == 'JuliaLang/julia' && github.event.pull_request.draft != true }}
```